backend/app/services/parse_csv.py

```diff
@@ -0,0 +1,37 @@
+from app.models import Document, Corpus
+
+def parse_csv(csv_reader, title):
+    doc_list = []
+    for data in csv_reader:
+        new_doc = None
+        if Document.objects.filter(title=data['title']).count() == 0:  # If the data isn't already an instance in the Document model:
+            new_doc = Document.objects.create_document(title=data['title'], text=data['transcript'], author=data['speaker_1'], year=data['recorded_date'][:4])
```

Comment:
I think that this is perfectly suitable for right now, since we're working with a prescribed set of test data (coming from Twitter). But ultimately, we'll likely want to allow the user to include the keys that we'll use to populate these properties. That is: the input dataset might use "publication_date" or something instead of "recorded_date", and we'll need to accommodate that possibility. For now, I'll recommend simply leaving a comment indicating that these keys match the expected Twitter dataset.